raiwidgets/raiwidgets/responsibleai_dashboard_input.py

```diff
@@ -48,7 +48,8 @@ def __init__(
             self.dashboard_input.cohortData = []
 
         self._feature_length = len(self.dashboard_input.dataset.feature_names)
-        if hasattr(analysis, ManagerNames.ERROR_ANALYSIS):
+        if (hasattr(analysis, ManagerNames.ERROR_ANALYSIS) and
+           analysis.error_analysis is not None):
```

Comment:
I think this should actually not be necessary with my latest PR now:
https://msdata.visualstudio.com/Vienna/_git/responsibleai/pullrequest/1039234
that creates EA even for object detection now